content/ko/docs/topics/kubernetes_distros.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "쿠버네티스 배포판 가이드"
+description: "특정 쿠버네티스 환경에서 헬름 사용에 대한 정보 정리."
+aliases: ["/docs/kubernetes_distros/"]
```

Comment:
Please remove this line. Aliases to English should not appear in translations, as that will redirect readers from English to a translation.

Reply:
@bridgetkromhout  Yes, I  will remove aliases as well as also in future translations.